invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/index.js

```diff
@@ -168,7 +168,11 @@ const LogoUploader = ({ community, defaultLogo, hasLogo, onError }) => {
             <Icon name="upload" />
             {i18next.t("Upload new picture")}
           </Button>
-
+          <label className="helptext">
+            {i18next.t('File must be smaller than {{limit}} MB.', {
+              limit: Math.ceil(logoQuota / 10**6)
```

Comment:
Is there a suggestion for converting this value on a more centralized way? I noticed that we have a function `humanReadableBytes` https://github.com/inveniosoftware/react-invenio-deposit/blob/master/src/lib/components/FileUploader/utils.js. but could not import it and use it here.

Reply:
It is just missing to be exported?